### DISCUSSION TITLE
Faktöriyel!

### DISCUSSION THREAD

**Discussion by EfeMertIrez** [sonsuzus/sonsuzus#411] [Soru Cevap]
Soru linki:https://www.hackerearth.com/practice/basic-programming/input-output/basics-of-input-output/practice-problems/algorithm/find-factorial/

Problem:

Size pozitif bir N tamsayısı verildi. Bu sayının Faktöriyelini bulup yazdırmanız gerekiyor. Pozitif bir N tam sayısının Faktöriyeli, 1'den N'ye kadar olan aralıktaki tüm sayıların çarpımını ifade eder. Bir sayının faktöriyeli hakkında daha fazla bilgiyi [burada ](https://en.wikipedia.org/wiki/Factorial)bulabilirsiniz.

Giriş Formatı:
Girişin ilk ve tek satırı, faktöriyelini bulmanız gereken sayıyı belirten tek bir N tamsayısını içerir.

Çıkış biçimi
N sayısının faktöriyelini gösteren tek bir satır çıktısı alın.

Kısıtlamalar
1 < N <10

**Comment by Epus0gi** [comment]
number = int(input("Lütfen faktöriyelini bulmak istediğiniz sayıyı giriniz...\n"))
factorial = 1
i = 1
 
if number >= 0:
    while i <= number:
        factorial *= i
        i += 1
    print(f"Girdiğiniz sayının faktöriyeli: {number}! = {factorial}")
else:
    print("Negatif sayıların faktöriyeli olmaz!")

**Comment by Sadik1Dev** [comment]
```
faktoriyel=1

n=int(input("lütfen bir ve on arasında bir sayı girin: "))

if n <= 1 or n >= 10:
    n=int(input("lütfen bir ve on arasında bir sayı girin: "))

#print(n)
    
while n >=1:
    faktoriyel = faktoriyel*n
    #print(faktoriyel,n)
    n-=1
    
print (faktoriyel)
```
